crates/astria-sequencer/src/accounts/mod.rs

```diff
@@ -39,34 +37,19 @@ impl AddressBytes for Address {
 }
 
 impl AddressBytes for [u8; ADDRESS_LEN] {
-    fn address_bytes(&self) -> [u8; ADDRESS_LEN] {
-        *self
-    }
-}
-
-impl AddressBytes for SignedTransaction {
-    fn address_bytes(&self) -> [u8; ADDRESS_LEN] {
-        self.address_bytes()
+    fn address_bytes(&self) -> &[u8; ADDRESS_LEN] {
```

Comment:
There are a few instances where making this `[u8; ADDRESS_LEN]` seemed more elegant. We could just add another trait method `as_bytes` in addition to `bytes`?

Reply:
Fixed in [132bddc](https://github.com/astriaorg/astria/pull/1492/commits/132bddc7e5a700a5289c61dee5188628a929795b).